'quicktype-core' has missing dependency 'is-url'

`quicktype-core` depends on `is-url`, but there is no reference in its `package.json`.

### Reproducntion steps
1. `mkdir test`
1. `cd test`
1. `npm i quicktype-core`
1. `node -e "require('quicktype-core')"`
1. The following error occurs

```
internal/modules/cjs/loader.js:1032
  throw err;
  ^

Error: Cannot find module 'is-url'
```

--- a/build/quicktype-core/package.in.json
+++ b/build/quicktype-core/package.in.json
@@ -14,6 +14,7 @@
     },
     "dependencies": {
         "collection-utils": "^1.0.1",
+        "is-url": "^1.2.4",
         "js-base64": "^2.4.3",
         "pako": "^1.0.6",
         "pluralize": "^7.0.0",

